checkpoint_ngfw/plugin.spec.yaml

```
@@ -728,29 +717,24 @@ actions:
         type: boolean
         required: true
 
-  add_host_to_network_group:
-    title: Add Host to Network Group
-    description: Add a host to a network group
+  add_address_object_to_group:
+    title: Add Address Object to Group
+    description: Add an address object (host) to a group
     input:
-      group_name:
-        title: Group Name
+      group:
+        title: Group
         description: Name of the group to add this object to
         required: true
         type: string
         order: 1
-      host_name:
+        example: "InsightConnect Block List"
+      address_object:
         title: Host Name
-        description: The host to add to the network group, usually the IP address
+        description: The name of the host to add, usually the IP address
```

I'm wondering if `usually the IP address` will be confusing - might indicate that IPs are find to use (without them being address objects of the same name)

```suggestion
        description: The name of the host object to add
```